__init__.py

```diff
@@ -516,6 +520,8 @@ def _cancel_all_timers(self):
             self.speak_dialog("cancelled-single-timer")
         else:
             self.speak_dialog("cancel-all", data={"count": len(self.active_timers)})
+        if self.expired_timers:
+            self.bus.emit(Message("timer.stopped-expired"))
```

Comment:
Can we make this `skill.timer.stopped-expired` so that we keep all Skills messages namespaced together?

I think this helps makes it clear that it's the Timer Skill and not some other timer happening somewhere in Mycroft's vast code base.